go/flags/endtoend/mysqlctl.txt

```diff
@@ -40,7 +40,7 @@ Flags:
       --db-credentials-vault-tls-ca string                          Path to CA PEM for validating Vault server certificate
       --db-credentials-vault-tokenfile string                       Path to file containing Vault auth token; token can also be passed using VAULT_TOKEN environment variable
       --db-credentials-vault-ttl duration                           How long to cache DB credentials from the Vault server (default 30m0s)
-      --db_charset string                                           Character set used for this tablet. (default "utf8mb4")
+      --db_charset string                                           Character set/collation used for this tablet. Make sure to configure this to a charset/collation supported by the lowest MySQL version in your environment. (default "utf8mb4")
```

Comment:
I think we probably want to drop the `Make sure` bit since for anyone running MySQL 8 and later (which is the only supported version), the default is fine and doesn't need configuring.

Reply:
What are you proposing? Drop the entire second sentence? Can you post that as a suggestion so that it is clearer?

Reply:
@dbussink FWIW, I kept the wording vague not mentioning $oldUnsupportedVersion. Just for the case that oracle would add a newer version of the Unicode character set in a future major upgrade… In that case this would become relevant again… but if the consensus is to remove the sentence, I can do that…

Reply:
I'd say we can drop the entire second sentence?  Maybe something like:

`Character set or collation name used by this tablet for the connection to MySQL`

> Just for the case that oracle would add a newer version of the Unicode character set in a future major upgrade…

I doubt this would happen, but they do have 4 chances to do this :wink:. See:

```
SELECT id, COLLATION_NAME FROM INFORMATION_SCHEMA.COLLATIONS order by id;
...
| 249 | gb18030_bin                 |
| 250 | gb18030_unicode_520_ci      |
| 255 | utf8mb4_0900_ai_ci          |
...
```

Giving this is a one byte field in the MySQL handshake protocol, only 251, 252, 253 & 254 are not used yet. You can't use any of the names with an `Id > 255` here anyway because of that limitation. 

But I also think that if that time comes, we can further clarify it with the situation that is then current?

Reply:
I say let's merge this for now and if and when things change we can revisit.